### DISCUSSION TITLE
How to inspect a container?

### DISCUSSION THREAD

**Discussion by yaqwsx** [containers/podman-py#79] [Q&A]
Hello, I have just installed v3.1.1.3 and I would like to inspect a container. Although I see an `inspect` function in https://github.com/containers/podman-py/blob/master/podman/containers/__init__.py, it is not available on my local installation.

What do I miss? How can I inspect a container?

**Comment by jwhonce** [comment]
@yaqwsx 

```
with PodmanClient() as client:
    container = client.containers.get(container_id)
    ...
```

**Reply by yaqwsx** [reply]
I probably miss something, but how do I get the same values for the container that `podman inspect` produces? Could you show me e.g., how to access the attribute "ExitCode", "State" or "StartedAt" from the snippet above?

**Reply by jwhonce** [reply]
@yaqwsx Sure.

```
with PodmanClient() as client:
    container = client.containers.get(container_id)
    print("Id:", container.id)
    print("ExitCode:", container.attrs["State"]["ExitCode"])
    print("State:", container.attrs["State"])
    print("StartedAt:", container.attrs["State"]["StartedAt"])
```

https://docs.podman.io/en/latest/_static/api.html#operation/ContainerInspectLibpod provides the JSON payload from the API call which is loaded into the `attrs` dictionary. The podman-py documentation will be hosted on-line in the future.
